client/src/components/prospects/Checkbox.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import {Checkbox as MaterialCheckbox} from "@material-ui/core";
+
+export default function Checkbox(props) {
+  const [checked, setChecked] = React.useState(props.checked);
+
+  const handleChange = (event) => {
+    setChecked(event.target.checked);
+
+    // Make sure the clickEvent prop was set first.
+    if (props['clickEvent']){
+      // Then call the callback, adding status of the checkbox instead of an event.
+      props.clickEvent(event.target.checked)
+    }
+  };
```

Comment:
I would suggest importing PropTypes and using that to ensure that the correct props are passed in - that way the component will throw an error if the required props are not passed in. 

I believe it's implemented in CampaignStep.js and GmailDialog.jsx, if you want to take a look at an example.